test/specs/chart/RadialBarChartSpec.js

```diff
@@ -1,218 +0,0 @@
-    expect(wrapper.find('.recharts-radial-bar-sector').length).to.equal(7);
-  });
-
-  it('Renders 7 sectors in simple RadialBarChart', () => {
-    const label = { orientation: 'outer' };
-    const wrapper = render(
-      <RadialBarChart
-        width={500}
-        height={300}
-        cx={150}
-        cy={150}
-        innerRadius={20}
-        outerRadius={140}
-        barSize={10}
-        data={data}
-      >
-        <RadialBar
-          startAngle={0}
-          endAngle={180}
-          minAngle={15}
-          label={label}
-          background
-          clockWise
-          dataKey="uv"
-          isAnimationActive={false}
-        />
-      </RadialBarChart>,
-    );
-    expect(wrapper.find('.recharts-radial-bar-sector').length).to.equal(7);
-  });
-
-  it('Renders 7 customized shape when shape is set to be a react element', () => {
-    const label = { orientation: 'outer' };
-    const Shape = props => <Sector {...props} className="customized-shape" />;
-    const wrapper = render(
-      <RadialBarChart
-        width={500}
-        height={300}
-        cx={150}
-        cy={150}
-        innerRadius={20}
-        outerRadius={140}
-        barSize={10}
-        data={data}
-      >
-        <RadialBar
-          shape={<Shape />}
-          minAngle={15}
-          label={label}
-          background
-          clockWise
-          dataKey="uv"
-          isAnimationActive={false}
-        />
-      </RadialBarChart>,
-    );
-    expect(wrapper.find('.customized-shape').length).to.equal(7);
-  });
-
-  it('Renders 7 customized shape when shape is set to be a function', () => {
-    const label = { orientation: 'outer' };
-    const renderShape = props => <Sector {...props} className="customized-shape" />;
-    const wrapper = render(
-      <RadialBarChart
-        width={500}
-        height={300}
-        cx={150}
-        cy={150}
-        innerRadius={20}
-        outerRadius={140}
-        barSize={10}
-        data={data}
-      >
-        <RadialBar
-          shape={renderShape}
-          minAngle={15}
-          label={label}
-          background
-          clockWise
-          dataKey="uv"
-          isAnimationActive={false}
-        />
-      </RadialBarChart>,
-    );
-    expect(wrapper.find('.customized-shape').length).to.equal(7);
-  });
-
-  it('Renders 7 legend item when add a Legend element', () => {
-    const wrapper = render(
-      <RadialBarChart
-        width={500}
-        height={300}
-        cx={150}
-        cy={150}
-        innerRadius={20}
-        outerRadius={140}
-        barSize={10}
-        data={data}
-      >
-        <RadialBar minAngle={15} background clockWise dataKey="uv" isAnimationActive={false} />
-        <Legend />
-      </RadialBarChart>,
-    );
-    expect(wrapper.find('.recharts-legend-wrapper').length).to.equal(1);
-    expect(wrapper.find('.recharts-legend-item').length).to.equal(7);
-  });
-
-  // it('Renders tooltip when add a Tooltip element', () => {
```

Comment:
Please do try to get this test working as well. The fact that it is commented out merely signifies a TODO.